### PR TITLE
Updates and fixes

### PR DESCRIPTION
**Pull request by prasser**

### CODE REVIEW

File: org.deidentifier.arx.appdata.xml [hunk 1]
```diff
@@ -23,12 +23,14 @@ ARX is able to handle large datasets on commodity hardware and it features an in
...
   <screenshots>
     <screenshot type="default">
-      <caption>jExifToolGUI on GNU/Linux</caption>
-      <image type="source">https://arx.deidentifier.org/wp-content/uploads/2015/12/front_page_screen_2.png</image>
+      <caption>ARX on GNU/Linux</caption>
+      <image type="source">https://raw.githubusercontent.com/flathub/org.deidentifier.arx/master/arx-on-linux.png</image>
     </screenshot>
   </screenshots>
   <releases>
     <release version="v3.9.1" date="2022-11-27"/>
```
Comment (muelli) [inline org.deidentifier.arx.appdata.xml:31]: this tag is closed twice

File: org.deidentifier.arx.appdata.xml [hunk 2]
```diff
@@ -23,12 +23,14 @@ ARX is able to handle large datasets on commodity hardware and it features an in
   </description>
   <screenshots>
     <screenshot type="default">
-      <caption>jExifToolGUI on GNU/Linux</caption>
-      <image type="source">https://arx.deidentifier.org/wp-content/uploads/2015/12/front_page_screen_2.png</image>
+      <caption>ARX on GNU/Linux</caption>
+      <image type="source">https://raw.githubusercontent.com/flathub/org.deidentifier.arx/master/arx-on-linux.png</image>
```
Comment (muelli) [inline org.deidentifier.arx.appdata.xml:27]: yeah, this has not been merged yet.
Also, this is not a stable URL which might make building in the future harder.
Using https://raw.githubusercontent.com/flathub/org.deidentifier.arx/b416ecb7e9b4a21d7c9af8dd6064bd9a0b8a8d0a/arx-on-linux.png should work better for both reasons

Comment (prasser) [reply org.deidentifier.arx.appdata.xml]: Thanks. Done.

**Comment by muelli** [comment]
heh.
`error: /srv/buildbot/worker/build-x86_64-5/build/builddir/export/share/icons/hicolor/scalable/apps/org.deidentifier.arx.svg is not a valid icon: Expected a square icon but got: 132x133`

yeah, this is annoying :-|

**Comment by muelli** [comment]
I also notice it's building for  aarch64.
It's not supposed to, right? I thought it's x86_64 only.
The filename of our flathub.json is probably wrong. I think I need to remove the leading dot.

**Comment by muelli** [comment]
excellent! Thanks!

**Pull request merged**